doc/development/squid.md

```diff
@@ -71,182 +85,218 @@ Parameters:
 ```
-oceanBalance= getOceanBalance(address)
+oceanBalance= ocean.getOceanBalance(address)
 ```
 
 * **getEtherBalance** - SYNC. The only parameter required is an account address. Returns the Ether balance for that account.
 ```
-ethBalance= getEtherBalance(address)
+ethBalance= ocean.getEtherBalance(address)
 ```
 
 * **getBalance** - SYNC. The only parameter required is an account address. Returns the Ocean tokens and Ether balance for that account.
 ```
-balance= getBalance(address)
+balance= ocean.getBalance(address)
 ```
 
-
-### Tokens
-
-#### Functions
-
-* **requestTokens** - SYNC. Request a number of Ocean Tokens. Returns a boolean to know if everything was right.
-
+* **requestTokens** - SYNC. Request a number of Ocean Tokens for an address. Returns a boolean to know if everything was right.
 ```
-result= requestTokens(amountTokens)
+result= ocean.requestTokens(account_address, amountTokens)
 ```
 
+* **searchAssets** - SYNC. Given a search query, returns a list of the DDO's matching with that query
+```
+array[assetDID]= ocean.searchAssets(searchQuery)
+```
 
-### Decentralized Identifiers (DID)
+* **getTrader** - ASYNC. Given an account address create a trader object
+```
+trader= ocean.getTrader(consumerAddress)
+```
 
-#### Functions
+* **searchOrders** - SYNC. Return a list of orders by search query. **Nice to Have**.
+```
+array[orderId]= ocean.searchOrders(searchQuery)
+```
 
 * **generateDID** - SYNC. Generates a specific DID with a random id based on the given.
 
 ```
-did= generateDID(seed)
+did= ocean.generateDID(seed)
 ```
 
 * **resolveDID** - SYNC. Given a DID the method returns the DDO associated to it.
 ```
-ddo= resolveDID(did)
+ddo= ocean.resolveDID(did)
 ```
 
-* **createDDO** - SYNC. Given the public keys, services and metadata information, creates a DDO and a DID.
+* **registerAsset** - ASYNC. High-level method publishing the metadata off-chain and registering the Service Agreement on-chain. It orchestrate the `publishAssetMetadata` and `publishServiceAgreement`, and creating a `DDO` methods.
 ```
-ddo= createDDO(pubKeys, services, metadata)
+Asset= ocean.registerAsset(metadata, services)
 ```
 
+* **getAsset** - ASYNC. Get an asset based on it's DID or assetId
+```
+Asset= ocean.getAsset(assetDID | assetId )
+```
 
+* **getServiceAgreement** - SYNC. Returns as serviceAgreement object based from the service agreement id (Publisher). **Nice to Have**.
+```
+serviceAgreement= ocean.getServiceAgreement(serviceAgreementId)
+```
 
-## Assets
+* **getOrder** - ASYNC. Get an order based on a orderId
+```
+Order= ocean.getOrder(orderId)
+```
 
-### Metadata
 
-#### Functions
+### Asset 
 
-* **register** - ASYNC. High-level method publishing the metadata off-chain and registering the Service Agreement on-chain. It orchestrate the `publishMetadata` and `publishServiceAgreement` methods.
+* **getDDO** - ASYNC. Return the created DDO used by this asset.
```

Comment:
this could be SYNC. getDDO should return the DDO & Metadata from OceanDB

Reply:
I've corrected the non block chain/async items to SYNC